### ISSUE TITLE
Re-trained weight of feature extractor model for vehicle instead of MARS

### ISSUE THREAD

**Issue by sangnguyenz** [Akhtar303/Vehicle-Detection-and-Tracking-Usig-YOLO-and-Deep-Sort-with-Keras-and-Tensorflow#1]
Thanks for your implementation for vehicle detection and tracking by YOLO&DeepSort. I had a question that why you still use the pre-trained weight on MARS dataset (pedestrian dataset) to compare appearance between vehicles? 

Do you think about training on vehicle dataset with proposed network (cosine metric learning)?

Let me know if you have any improvement. Thank you!

**Comment by Akhtar303** [comment]
@sangnguyenz  yes you can try it.

**Comment by sangnguyenz** [comment]
I can see in the model_data folder contain frozen_inference_graph.pb. Can you tell me if this weight was used for what?
Thank you for your instant reply.

**Comment by quocnhat** [comment]
@sangnguyenz have you retrained the weight to extract vehicle embedding? I want to try but I have no idea how to do it. please give me some hinds, thank you

**Comment by sangnguyenz** [comment]
@quocnhat Currently I haven't continued to work on this project, so that I haven't tried anything since September. Sorry for can not help you with this.
